app.js

```diff
@@ -40,7 +40,7 @@ const storage = getStorage();
 
 /* c8 ignore start */
 const appVersion =
-  process.env.GAE_VERSION === 'production' ?
+  process.env.NODE_ENV === 'production' ?
```

Comment:
Is there any particular reason we can't just let this code read package.json unconditionally? Without a staging environment there's just local and testing and we won't confuse ourselves there I think.

Reply:
The rest of our code relies on appVersion being set to `Dev` to determine whether or not we're running a development build, so that it may report as such.  For example, if a PR is submitted from the dev version, the PR message will report as such to indicate we probably shouldn't merge it.  On that note, technically speaking, the version reported by `package.json` won't be accurate on dev builds since we update the version number just before a release.

I had the idea of using the shortened commit hash as the version number when running a development build, but I didn't want to worry about implementing such logic at the time.  How would that sound to you, though?

Reply:
I see, so we do use this value for stuff beyond the UI. Let's keep this simplification to the current scope then.

Using `git describe` would make sense I think, but in a separate PR if so.